frontend/src/queries/nodes/InsightViz/EditorFilters.tsx

```diff
@@ -30,13 +32,28 @@ import { TrendsFormulaLabel } from './TrendsFormulaLabel'
 import { TrendsFormula } from './TrendsFormula'
 import { Breakdown } from './Breakdown'
 import { getBreakdown, getDisplay } from './utils'
+import { PathsEventsTypesDataExploration } from 'scenes/insights/EditorFilters/PathsEventTypes'
+import {
+    PathsTargetEndDataExploration,
+    PathsTargetStartDataExploration,
+} from 'scenes/insights/EditorFilters/PathsTarget'
+import { PathsExclusionsDataExploration } from 'scenes/insights/EditorFilters/PathsExclusions'
+import { PathsWildcardGroupsDataExploration } from 'scenes/insights/EditorFilters/PathsWildcardGroups'
+import { PathsAdvancedDataExploration } from 'scenes/insights/EditorFilters/PathsAdvanced'
+import { PayGateMini } from 'lib/components/PayGateMini/PayGateMini'
 
 export interface EditorFiltersProps {
     query: InsightQueryNode
     setQuery: (node: InsightQueryNode) => void
 }
 
 export function EditorFilters({ query, setQuery }: EditorFiltersProps): JSX.Element {
+    const { user } = useValues(userLogic)
+    const availableFeatures = user?.organization?.available_features || []
```

Comment:
I've seen this `user?.organization?.available_features` enough times that we should likely just have a selector `availableFeatures` on `userLogic` by now :D